### ISSUE TITLE
Player moves faster while strafing.

### ISSUE THREAD

**Issue by Obambulate** [chrisj42/minicraft-plus-revived#172]
The player moves faster while strafing. There could also be a running mechanic as well.

**Comment by Dioarya** [comment]
I think a possible fix for this is to divide the `xd` and `yd` by 2 when moving diagonal

**Comment by Makkkkus** [comment]
You should divide it by 1.2 or 1.45 if I remember correctly

**Comment by El-Virus** [comment]
No, actually we should divide it by 2 as we have 2 values, the player will run at 1.4 or 1.7 the walking speed.

**Comment by Makkkkus** [comment]
Put on hold since #199 needs to be done first.
